build.sh

```diff
@@ -284,6 +284,12 @@ function copyResources {
 	
 	# Remove .jsx files - we'll deal with those in gulp
 	find "$browser_builddir" -type f -name "*.jsx" -delete
+
+	# Copy SingleFile submodule code
+	mkdir -p "$browser_builddir/lib/SingleFile/extension/lib"
+	cp -r "$SRCDIR/zotero/resource/SingleFile/extension/lib/single-file" \
+		"$browser_builddir/lib/SingleFile/extension/lib/single-file"
+	cp -r "$SRCDIR/zotero/resource/SingleFile/lib" "$browser_builddir/lib/SingleFile/lib"
```

Comment:
The multiple submodules was my mistake. It is only included once now. The reason for maintaining the directory structure was because I was using their injection script for the reasons described below. If we use our own list of files then yes we can absolutely change the name to something shorter.